app/models/permit_application.rb

```diff
@@ -415,6 +417,29 @@ def energy_step_code_required?
     custom_requirements.empty? || custom_requirements.any? { |r| r.energy_step_required || r.zero_carbon_step_required }
   end
 
+  def self.count_by_jurisdiction_and_status
+    Jurisdiction.all.map do |j|
+      row = PermitApplication.with_submitter_role.where(jurisdiction: j).group(:status).count
+
+      {
+        "draft" => (row["new_draft"] || 0) + (row["revisions_requested"] || 0), # nil defaults to 0
+        "submitted" => (row["newly_submitted"] || 0) + (row["resubmitted"] || 0),
+      }.merge({ name: j.qualified_name })
+    end
+  end
+
+  def self.count_by_jurisdiction_and_status
+    Jurisdiction.all.map do |j|
+      row =
+        PermitApplication.joins(:submitter).where(jurisdiction: j, users: { role: "submitter" }).group(:status).count
+
+      {
+        "draft" => (row["new_draft"] || 0) + (row["revisions_requested"] || 0), # nil defaults to 0
+        "submitted" => (row["newly_submitted"] || 0) + (row["resubmitted"] || 0),
+      }.merge({ name: j.qualified_name })
+    end
+  end
```

Comment:
duplicate method

Reply:
this needs to be removed altogether as there is now an advanced version of this on develop